set.mm

```diff
@@ -56004,6 +56036,24 @@ empty set when it is not meaningful (as shown by ~ ndmfv and ~ fvprc ).
       ( wf wss fss syl2anc ) ABCEHCDIBDEHFGBCDEJK $.
   $}
 
+  ${
+    fssdmd.f $e |- ( ph -> F : A --> B ) $.
+    fssdmd.d $e |- ( ph -> D C_ dom F ) $.
+    $( Restricting the domain of a mapping, deduction form.  (Contributed by
```

Comment:
There is no restriction here (and in the next theorem). Maybe "Expressing that a class is a subclass of the domain of a function expressed in maps-to notation ?"

Reply:
Comments changed accordingly.